src/components/vms/hostvmslist.jsx

```diff
@@ -126,6 +141,13 @@ const HostVmsList = ({ vms, config, ui, storagePools, actions, networks, onAddEr
             <Page>
                 <PageSection>
                     <Gallery className="ct-cards-grid" hasGutter>
+                        {!virtualizationEnabled && <Alert variant="warning" title="Warning: Virtualization is disabled in the firmware" component="h5">
```

Comment:
replace title with: _("Hardware virtualization extensions are disabled in BIOS")

I am not sure if we want to add the steps for fixing this inline in cockpit. @garrett @martinpitt what do you think? 
It's quite of an edge scenario, as all new servers will come with virtualization enabled. And it's a lot of text to translate.

Reply:
Thanks for the feedbacks.

Linking it to the FAQ sounds better to me because it avoids repeating instructions already present in the FAQ.

I tested it by rebooting and toggling the settings in BIOS a few times. It was not ideal. I'll try `rmmod kvm_intel` . Thanks.

Reply:
Implemented the suggested changes.

@martinpitt `rmmod kvm_intel` doesn't seem to work. At least, it doesn't have an immediate effect.

Reply:
`rmmod kvm_intel` would't help as the kernel populates `/proc/cpuinfo` and `kvm_intel` is just a way to use the virtualisation extensions of the CPU, it could as well be Xen or Virtualbox.